build-system/setup/__init__.py

```diff
@@ -477,6 +478,52 @@ def install_python_requirements ():
 
 
 
+"""
+==============================================================================
+Name: install_max_package
+==============================================================================
+"""
+
+def install_max_package ():
+   if platform.system () == 'Linux':
+      return
+
+   max8_packages_path = os.path.join (Path.home (), 'Documents', 'Max 8', 'Packages')
+   src_path = os.path.join (PATH_ROOT, 'max', 'Eurorack-blocks')
+   dst_path = os.path.join (max8_packages_path, 'Eurorack-blocks')
+
+   def is_link (path):
+      try:
+         return os.path.islink (path) or bool (os.readlink (path))
+      except OSError:
+         return False
+
+   def create_erbb_max_package_symlink ():
+      if platform.system () == 'Windows':
+         import _winapi
+         _winapi.CreateJunction (src_path, dst_path)
+      else:
+         os.symlink (src_path, dst_path)
+
+   if os.path.exists (max8_packages_path) and os.path.isdir (max8_packages_path):
+      if os.path.exists (dst_path):
+         if os.path.samefile (dst_path, src_path):
+            print ('"Eurorack-blocks" Max package already installed.')
+
+         elif is_link (dst_path):
+            os.remove (dst_path)
+            create_erbb_max_package_symlink ()
+            print ('"Eurorack-blocks" Max package installed.')
+
+         else:
+            print (f'"Eurorack-blocks" file in max packages already exist.\nPlease remove "{dst_path}" and run `erbb setup` again.')
```

Comment:
Please prefix with warning:

```suggestion
            print (f'\033[33mwarning:\033[0m"Eurorack-blocks" file in max packages already exist.\nPlease remove "{dst_path}" and run `erbb setup` again.')
```